packages/@lwc/engine/src/framework/decorators/__tests__/wire.spec.ts

```diff
@@ -7,6 +7,7 @@
 import { compileTemplate } from 'test-utils';
 import { createElement, LightningElement } from '../../main';
 import wire from '../wire';
+import { registerDecorators } from '../register';
```

Comment:
We should be consistent and import `createElement`, `LightningElement`, and `registerDecorators` from the same path.